app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -42,6 +47,17 @@ def security_event_tokens
       @security_event_tokens ||= redis_client.read_events(timestamp: timestamp)
     end
 
+    def encrypted_security_event_log_result
+      json = security_event_tokens.to_json
+      decoded_key_der = Base64.strict_decode64(IdentityConfig.store.irs_attempt_api_public_key)
+      pub_key = OpenSSL::PKey::RSA.new(decoded_key_der)
+
+      IrsAttemptsApi::EnvelopeEncryptor.encrypt(
+        data: json, timestamp: timestamp,
+        public_key: pub_key
+      )
```

Comment:
my vote is for "all on the same line" or "all on different lines"
```suggestion
      IrsAttemptsApi::EnvelopeEncryptor.encrypt(
        data: json, timestamp: timestamp, public_key: pub_key
      )
```

Reply:
Good call, fixed in https://github.com/18F/identity-idp/pull/6776/commits/a4f3e02b0f813777a80f4795273c149bb46003f0